src/main/java/teammates/ui/controller/AdminSessionsPageAction.java

```diff
@@ -101,35 +97,35 @@ private ActionResult validateParametersAndCreateShowPageResultIfInvalid() {
             String endMin = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_ENDMINUTE);
             String timeZone = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_TIMEZONE);
 
-            zone = ZoneId.of(timeZone);
-            start = TimeHelper.parseLocalDateTimeForSessionsForm(startDate, startHour, startMin);
-            end = TimeHelper.parseLocalDateTimeForSessionsForm(endDate, endHour, endMin);
-
-            if (start.isAfter(end)) {
-                isError = true;
-                statusToUser.add(new StatusMessage("The filter range is not valid."
-                        + " End time should be after start time.", StatusMessageColor.DANGER));
-                statusToAdmin = "Admin Sessions Page Load<br>"
-                        + "<span class=\"bold\"> Error: invalid filter range</span>";
-                prepareDefaultPageData(start, end);
-                return createShowPageResultWithDataInitialization();
-            }
+            this.rangeStart = TimeHelper.parseLocalDateTimeForSessionsForm(startDate, startHour, startMin);
+            this.rangeEnd = TimeHelper.parseLocalDateTimeForSessionsForm(endDate, endHour, endMin);
+            this.timeZone = ZoneId.of(timeZone);
```

Comment:
Parsing these 3 things may result in errors as well, so `rangeStart`, `rangeEnd` may be `null` and `ZoneId.of(timeZone)` may throw an Exception